modeling/inference_model.py

```diff
@@ -6,6 +6,10 @@
 from logger import logger
 
 import torch
+try:
```

Comment:
Does this do anything? similar possibly unneeded imports in many other files

Reply:
ipex adds the actual XPU support. Torch on it's own doesn't support XPU.

Reply:
Ah, so just the import patches it. Does it need to be done in each file that imports torch? I thought python shared states of modules within the same process

Reply:
Well Intel says import it immediately after Torch but it seems to be working fine when imported in the main .py file.